src/serialized-analysis.ts

```diff
@@ -0,0 +1,425 @@
+export interface SerializedAnalysis { packages: Package[]; }
+
+export interface Package {
```

Comment:
I don't think we need `Package`. An `elements.json` file should already be in a package, next to a `bower.json` and/or `package.json`. No need to duplicate that info here.